lib/s3_helper.rb

```diff
@@ -98,11 +98,13 @@ def remove(s3_filename, copy_to_tombstone = false)
     s3_filename.prepend(multisite_upload_path) if Rails.configuration.multisite
     delete_object(get_path_for_s3_upload(s3_filename))
   rescue Aws::S3::Errors::NoSuchKey
+  rescue Aws::S3::Errors::NotFound
```

Comment:
You can list the exceptions you want to `rescue` separated by a comma `,`

```suggestion
  rescue Aws::S3::Errors::NoSuchKey, Aws::S3::Errors::NotFound
```

Reply:
Ahah just left the same comment :grin: